compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.td

```diff
@@ -57,8 +57,21 @@ def IREEGPU_LoweringConfigAttr :
         "The configured fields, including tiling levels">:$attributes
   );
   let extraClassDeclaration = [{
-    /// Helper to retrieve a target mma intrinsic if present.
+    /// Helper to retrieve/set a target mma intrinsic.
     ::mlir::iree_compiler::IREE::GPU::MmaInterfaceAttr getMmaKind() const;
+    static void setMmaKind(MLIRContext *context,
+                          SmallVectorImpl<NamedAttribute> &attrs,
+                          ::mlir::iree_compiler::IREE::GPU::MmaInterfaceAttr kind);
+
+    /// Helper to retrieve/set a target subgroup M/N counts.
+    std::optional<int64_t> getSubgroupMCount() const;
+    std::optional<int64_t> getSubgroupNCount() const;
+    static void setSubgroupMCount(MLIRContext *context,
+                                   SmallVectorImpl<NamedAttribute> &attrs,
+                                   int64_t subgroup_m_count);
+    static void setSubgroupNCount(MLIRContext *context,
+                                   SmallVectorImpl<NamedAttribute> &attrs,
+                                   int64_t subgroup_n_count);
```

Comment:
I'm a bit scared that these will be confused with subgroup tiling level

Reply:
I thought of using MMAScheduleAttr originally, but since PackToIntrinsics seem to use getMmaKind https://github.com/iree-org/iree/blob/9c85e30df30d6efcf68a7a1b594e89322bd6085d/compiler/src/iree/compiler/Codegen/Dialect/GPU/Transforms/PackToIntrinsics.cpp#L41-L43 
I was thinking it'd be awkward to have both mma_kind and MMAScheduleAttr live in lowering_config. Alternatively, we can also replace mma_kind with MMA_Schedule while modifying mma_schedule_attr's `subgroup_m_count` and `subgroup_n_count` as optional attrs

Reply:
Alternatively, I can start using the subgroup_tiling_level, but that would mean I infer subgroup_tiling_level from subgroup _m_count and subgroup_n_count in KernelConfig, and then I infer the subgroup_m_count and subgroup_n_count in ConfigureTensorLayout? haha

Reply:
I'm fine with either using a mma_schedule getter/setter or using a subgroup_m_count/subgroup_n_count getter/setter. Let's not do the subgroup/thread tiling level configuration right now, as that will go make this NFC PR into something else that we will have to benchmark to see if it still is good.

Reply:
SG, if it's alright with you guys, I'd like to keep it as is then. Since we are planning to change this to `subgroup tiling level` in the future anyways, and this form is probably easier to modify to that form(i.e `subgroup tiling level`).

Reply:
Added a couple TODOs to replace/remove this with subgroup_level_tiling, hopefully that makes it a bit clearer to the readers of the code.